src/modules/auth/index.js

```diff
@@ -267,7 +267,19 @@ export default class AuthModule extends Module {
 		// if session has expired
 		if (!isUserInfoUpdated && oldUserInfo) {
 			oldUserInfo = null;
-			that.App.addAlert("danger", "ASABAuthModule|Your session has expired", 3600 * 1000, true);
+			that.App.addAlert("danger", "ASABAuthModule|You have been logged out due to inactivity.", 3600 * 1000, true);
+			if (that.App.Store != null) {
```

Comment:
@Aringoaway Shouldnt be the queryselectors wrapped in this condition? AS per my knowledge from your code, some of the features will not be disabled is (for any reason) the Store wont be available